### ISSUE TITLE
Slider Property for if drag is coming from PDOM

### ISSUE THREAD

**Issue by zepumph** [phetsims/sun#536]
From a few weeks ago dev meeting, in OHMS_LAW/SliderUnit, there is code that is keeping track of if the drag is from alternative input in the PDOM. This should be generalized. Likely to AccessibleSlider, but also perhaps to `AccessibleValueHandler`.

@jessegreenberg in AccessibleValueHandler, I feel like there is less of a notion of a "drag" or complete interactio that spans multiple inputs. Though the slider code that manages keyboard keys' state is in AccessibleValueHandler, would a Property like `sliderDraggingByKeyboardProperty` be more appropriate in one vs the other?

**Comment by jessegreenberg** [comment]
I think that this should go into AccessibleValueHandler because it seems like if we want this information for sliders we will probably want it for number spinners and number pickers too.

I am not sure I understand how what will be added will be used yet. Do we need to add something specifically for a keyboard or do we want knowledge of when any kind of discrete increment/decrement occurs from alternative input?

The usage in ohms-law/SliderUnit is specific to keyboard but maybe it should work for other screen reader gestures too.

**Comment by zepumph** [comment]
>or do we want knowledge of when any kind of discrete increment/decrement occurs from alternative input?

I guess I don't understand how this could be set to a Property. It feels similar to how we only get the "click" event from buttons with a screen reader. There is no async time step between the start of an input event and when it finishes. How would we, from an input/change event, toggle this Property any more than just at the beginning and end of the listener, like (pseudo code):

```js

        handleInput( event ) {
          if ( this._enabledProperty.get() && !this.blockInput ) {
                  this.changingFromA11yProperty.set(true);

                  // do the actual listener stuff

                  this.changingFromA11yProperty.set(false);
          }
        },

```

Perhaps that is all that is actually needed.

**Comment by zepumph** [comment]
This may be a duplicate of https://github.com/phetsims/sun/issues/536.

**Comment by zepumph** [comment]
I now think that this is a duplicate of https://github.com/phetsims/scenery/issues/1016. Over there we are trying to add more general logic to `DragListener` instead of just to a slider. Closing
